src/transformers/modeling_utils.py

```diff
@@ -2188,6 +2189,19 @@ def from_pretrained(
         subfolder = kwargs.pop("subfolder", "")
         commit_hash = kwargs.pop("_commit_hash", None)
         variant = kwargs.pop("variant", None)
+        kwargs.pop("peft_adapter_name", None)
+        peft_adapter_model_id = kwargs.pop("_peft_adapter_model_id", None)
+
+        is_adapter_file = False
+        if is_peft_available():
+            is_adapter_file = peft_adapter_model_id is not None
+
+        if is_adapter_file:
+            logger.info(
+                "Found adapter file at {}. Automatically loading adapter model using the base model from {}".format(
```

Comment:
Please use f-strings, we don't use `format` in Transformers :-)

Reply:
Thanks for the heads up !  should be fixed in [6385a94](https://github.com/huggingface/transformers/pull/24827/commits/6385a94e255bed4f89c56fed34eb290ca9b69b27)